packages/shell-api/src/replica-set.ts

```diff
@@ -18,15 +19,49 @@ export default class ReplicaSet extends ShellApiClass {
   constructor(mongo) {
     super();
     this._mongo = mongo;
-    const proxy = new Proxy(this, {
-      get: (obj, prop): any => {
-        if (!(prop in obj)) {
-          throw new MongoshUnimplementedError('rs not currently supported');
-        }
-        return obj[prop];
-      }
-    });
-    return proxy;
+  }
+
+  /**
+   *  rs.initiate calls replSetInitiate admin command.
+   *
+   * @param config
+   */
+  @returnsPromise
+  async initiate(config?: any): Promise<any> {
```

Comment:
```suggestion
  async initiate(config?: object): Promise<any> {
```

Reply:
Could also use `={}` for default argument to avoid the TS issue and prevent sending undefined to the server?